src/hooks/useGasLimit.ts

```diff
@@ -60,12 +64,23 @@ const useGasLimit = (
   const [gasLimit, gasLimitError, gasLimitLoading] = useAsync<BigNumber>(() => {
     if (!safeAddress || !walletAddress || !encodedSafeTx || !web3ReadOnly) return
 
-    return web3ReadOnly.estimateGas({
-      to: safeAddress,
-      from: walletAddress,
-      data: encodedSafeTx,
-      type: operationType,
-    })
+    return web3ReadOnly
+      .estimateGas({
+        to: safeAddress,
+        from: walletAddress,
+        data: encodedSafeTx,
+        type: operationType,
+      })
+      .then((gasLimit) => {
+        // Due to a bug in Nethermind estimation we need to increment 30% the gasLimit
```

Comment:
```suggestion
        // Due to a bug in Nethermind estimation on Gnosis Chain, we need to increment the gasLimit by 30%
```

Reply:
Technically the gas estimation issue can happen on any chain that relies on Nethermind. If we would use Nethermind on mainnet the same bug could happen. So the bug is indeed on Nethermind estimation

Reply:
Yeah, but the fix is only for GC, isn't it?